CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ option(FLB_RECORD_ACCESSOR     "Enable record accessor"       Yes)
 option(FLB_SIGNV4              "Enable AWS Signv4 support"    Yes)
 option(FLB_AWS                "Enable AWS support"            Yes)
 option(FLB_STATIC_CONF         "Build binary using static configuration")
+option(FLB_STATIC_BINARY       "Build static binary           No")
```

Comment:
Line must be changed to
option(FLB_STATIC_BINARY       "Build static binary"           No)